OpenRA.Mods.Common/Widgets/Logic/Ingame/ObserverStatsLogic.cs

```diff
@@ -573,12 +574,12 @@ static void AddPlayerFlagAndName(ScrollItemWidget template, Player player)
 
 		string AverageOrdersPerMinute(double orders)
 		{
-			return (world.WorldTick == 0 ? 0 : orders / (world.WorldTick / 1500.0)).ToString("F1");
+			return (world.WorldTick == 0 ? 0 : orders / (world.WorldTick / 1500.0)).ToString("F1", NumberFormatInfo.CurrentInfo);
 		}
 
 		string Vision(int revealedCells)
 		{
-			return Math.Ceiling(revealedCells / (float)world.Map.ProjectedCells.Length * 100).ToString("F0") + "%";
+			return Math.Ceiling(revealedCells / (float)world.Map.ProjectedCells.Length).ToString("P0", NumberFormatInfo.CurrentInfo);
```

Comment:
this makes it so vision score is at 100% at all times

Reply:
Nice spot thanks. Fixed by removing the `Math.Ceiling`.

Reply:
It was used so the percentage would not be at 0%

Reply:
Ok, adjusted to keep a `Math.Ceiling` in there.